git_info

```diff
@@ -0,0 +1,3 @@
+{
+  "test": "test"
+}
```

Comment:
What is this file for?

Reply:
:+1: this is a good idea

Reply:
Though frontstage-api also uses this method and the file does get overwritten when it is deployed

Reply:
Fair play, just think it's better to not have a default file for this. Similar to when we have env vars set to default values that can easily make their way to production.

Reply:
https://github.com/ONSdigital/ras-frontstage-api/pull/22added the change to frontstage-api